### PR TITLE
chipmunk: Add version 2.9.0

### PR DESCRIPTION
**Pull request by Slach**
chipmunk is a fast logfile viewer that can deal with huge logfiles (>10 GB). It powers a super fast search and is supposed to be a useful tool for developers who have to analyze logfiles.

### CODE REVIEW

File: bucket/chipmunk.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        [
+            "chipmunk.exe",
+            "chipmunk"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/esrlabs/chipmunk"
+    },
+    "autoupdate": {
+        "url": "https://github.com/esrlabs/chipmunk/releases/download/$version/chipmunk@$version-win64-portable.zip"
```
Comment (Ash258) [inline bucket/chipmunk.json:19]: 64bit

File: bucket/chipmunk.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.9.0",
+    "description": "fast logfile viewer that can deal with huge logfiles (>10 GB). It powers a super fast search and is supposed to be a useful tool for developers who have to analyze logfiles.",
+    "homepage": "https://github.com/esrlabs/chipmunk",
+    "license": "Apache 2.0",
```
Comment (Ash258) [inline bucket/chipmunk.json:5]: invalid spdx identifier

File: bucket/chipmunk.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/esrlabs/chipmunk/releases/download/2.9.0/chipmunk@2.9.0-win64-portable.zip",
```
Comment (Ash258) [inline bucket/chipmunk.json:6]: 64bit

File: bucket/chipmunk.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "bin": "chipmunk.exe",
+    "shortcuts": [
+        [
+            "chipmunk.exe",
+            "chipmunk"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/esrlabs/chipmunk"
+    },
```
Comment (Ash258) [inline bucket/chipmunk.json:17]: simplify

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/chipmunk.json (hunk 2 above)
Comment (Slach) [reply bucket/chipmunk.json]: changed to `Apache-2.0` https://spdx.org/licenses/Apache-2.0.html

File: bucket/chipmunk.json (hunk 3 above)
Comment (Slach) [reply bucket/chipmunk.json]: fixed

File: bucket/chipmunk.json (hunk 1 above)
Comment (Slach) [reply bucket/chipmunk.json]: fixed

File: bucket/chipmunk.json (hunk 4 above)
Comment (Slach) [reply bucket/chipmunk.json:22]: could you explain or suggest what exactly do you mean? I not  so familiar with scoop packages JSON syntax

Comment (Ash258) [reply bucket/chipmunk.json:22]: checkver: github

Comment (Slach) [reply bucket/chipmunk.json]: @Ash258 done, thanks a lot for suggestion

**Comment by Slach** [comment]
@Ash258, could you look to PR again?

**Review by Ash258 — APPROVED** [review]

**Comment by Slach** [comment]
@Ash258 what next step on pull request workflow? Can we merge these changes?

**Comment by Slach** [comment]
@Ash258 could you explain pull requests workflow for this project? maybe some links help me understand why this PR didn't merge?

**Pull request merged**